src/marlin_stubs/M876.cpp

```diff
@@ -23,6 +23,7 @@
 
 #if ENABLED(HOST_PROMPT_SUPPORT) && DISABLED(EMERGENCY_PARSER)
 
+		#include "../../lib/Marlin/Marlin/src/feature/safety_timer.h"
```

Comment:
This include already exists two lines below.

Reply:
Fixed - thank you @ojrik